apps/site/src/views/Resources/sections/Figures/Figures.module.scss

```diff
@@ -0,0 +1,95 @@
+@use "bootstrap-utils" as bootstrap;
+
+.hammerhead {
+	position: absolute;
+	visibility: hidden;
+	transform: translateY(10vh);
+}
+
+.birds {
+	position: absolute;
+	visibility: hidden;
+	width: 275px;
+}
+
+.palmTree {
+	position: absolute;
+	right: 0;
+	transform: translateY(90vh);
+	width: 30vw;
+
+	z-index: 2;
+	visibility: hidden;
+	height: auto;
+}
+
+.palmTreeShade {
+	position: absolute;
+	right: 0;
+	transform: translateY(calc(100vh + 50%));
+	width: 30vw;
+
+	z-index: 1;
+	visibility: hidden;
+	height: auto;
+}
+
+@include bootstrap.media-breakpoint-up(md) {
+	.palmTree {
+		visibility: visible;
+	}
+
+	.palmTreeShade {
+		visibility: visible;
+	}
+}
+
+@include bootstrap.media-breakpoint-up(lg) {
+	.hammerhead {
+		visibility: visible;
+		left: 5vh;
+	}
+
+	.birds {
+		visibility: visible;
+		right: 2vh;
+	}
+
+	.palmTree {
+		visibility: visible;
+	}
+
+	.palmTreeShade {
+		visibility: visible;
+	}
+}
+
+@media (min-width: 1200px) {
```

Comment:
suggestion (repeating): you can use the bootstrap media breakpoints for these sizes, it would just be xl and xxl